### ISSUE TITLE
Not compatible with Castle.Core 4.4.0

### ISSUE THREAD

**Issue by crystalgreen** [ninject/Ninject.Extensions.Interception#48]
I get the compiler error:

```
Message: System.TypeLoadException : Method 'CaptureProceedInfo' in type 'Ninject.Extensions.Interception.ProxyFactory.ObjectMethodsInvocation' from assembly 'Ninject.Extensions.Interception.DynamicProxy, Version=3.3.3.0, Culture=neutral, PublicKeyToken=c7192dc5380945e7' does not have an implementation.
```

I tracked it down to [ObjectMethodsInvocation](https://github.com/ninject/Ninject.Extensions.Interception/blob/master/src/Ninject.Extensions.Interception.DynamicProxy/ObjectMethodsInvocation.cs) which inherits from `IInvocation` which now requires implementation of [CaptureProceedInfo](https://github.com/castleproject/Core/blob/0f5bdb7c20d1d65aab60ddbf9fe8a92b56307b4b/src/Castle.Core/DynamicProxy/IInvocation.cs#L121).

**Comment by oviradoi** [comment]
I have the same problem.

**Comment by scott-xu** [comment]
Fixed in release 3.3.4
